SolidStateGroup/bullet-train-api
# fix: Reading role permissions generates 500 errors

Thanks for submitting a PR! Please check the boxes below:


## Changes

- Skip requests when the role organization is not sent
- Use new RBAC endpoint to retrieve and delete userRoles by user

## How did you test this code?

- Go to manage -> members -> Users
- Open the Network tab
- Click any User
- The error is no longer there
